frontend/src/components/Footer.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import "./Footer.scss";
+function Footer() {
+  return (
+    <footer>
+      <div className="LinearGradient"></div>
+      <div className="footer">
+        <div className="nav">
+          <p>Copyright © 2019</p>
```

Comment:
Generally speaking, you'd see an `<ul>` and some `<li>` tas instead of `<p>` (but that's just a recommendation)